src/Controls/src/Core/Handlers/Items/iOS/MauiCollectionView.cs

```diff
@@ -14,4 +16,18 @@ public override void ScrollRectToVisible(CGRect rect, bool animated)
 		if (!KeyboardAutoManagerScroll.IsKeyboardAutoScrollHandling)
 			base.ScrollRectToVisible(rect, animated);
 	}
+
+	[UnconditionalSuppressMessage("Memory", "MEM0002", Justification = IUIViewLifeCycleEvents.UnconditionalSuppressMessage)]
```

Comment:
Also the CarouselView is included in the HandlerDoesNotLeak. Looks fine pasing the test.